docs/_config.yml

```diff
@@ -0,0 +1,11 @@
+title: Solaar
+description: Linux Device manager for the Logitech Unifying Receiver.
+tagline: Linux Device manager for the Logitech Unifying Receiver.
+owner: pwr
+owner_url: https://github.com/pwr
+repository: pwr/Solaar
```

Comment:
We have moved! Could you update this?

Reply:
Woo! Super excited to see what this means for this project.